src/test/run-pass/lexer-crlf-line-endings-string-literal-doc-comment.rs

```diff
@@ -1,7 +1,7 @@
 // ignore-tidy-cr ignore-license
 // ignore-tidy-cr (repeated again because of tidy bug)
 // license is ignored because tidy can't handle the CRLF here properly.
-
+
```

Comment:
No! Not this file! This test is specifically supposed to use crlf! This is changing the blank lines from \r\n to \n!